docs/tutorials/python.md

```diff
@@ -0,0 +1,294 @@
+2. The `pixi_py` package is added as an editable dependency. This means that the package is installed in editable mode, so you can make changes to the package and see the changes reflected in the environment. In pixi, unlike other package managers this is explicitly stated in the `pyproject.toml` file. The main reason being so that you can choose into which environment this package should be included.
+
+Now let's add some optional dependencies:
+
+```toml
+[project.optional-depencencies]
+test = ["pytest"]
+
+# 3. Environments
+[tool.pixi.environments]
+default = { solve-group = "default" }
+test = { features = ["test"], solve-group = "default" }
+```
+
+We've added the optional dependencies to the `pyproject.toml` and this automatically creates a `feature`, which is a collection of `dependencies`, `tasks`, `channels` etc.
+These features can be combined
+
+3. We've created the `default` and `test` environments. The `default` environment is the default environment that is created when you run `pixi install`. The `test` environment is created from the optional dependencies in the `pyproject.toml` file. You can execute commands in this environment with e.g `pixi run -e test python`
+
+??? note "Solve Groups"
+    Solve groups are a way to group dependencies together. This is useful when you have multiple environments that share the same dependencies. This way you can solve dependencies together. E.g maybe `pytest` is a dependency that influences the dependencies of the `default` environment. By putting these in the same solve group, you assure that the versions in `test` and `default` are exactly the same versions.
+
+
+### Installation: `pixi install`
+
+Now let's `install` the project with `pixi install`:
+
+```shell
+$ pixi install
+✔ Project in /private/tmp/pixi_py is ready to use!
+```
+
+We now have a new directory called `.pixi` in the project root. This directory contains the environment that was created when we ran `pixi install`. The environment is a conda environment that contains the dependencies that we specified in the `pyproject.toml` file. We can also install the test environment with `pixi install -e test`. We can use these environments for executing code.
+
+We also have a new file called `pixi.lock` in the project root. This file contains the exact versions of the dependencies that were installed in the environment, across platforms.
+
+## What's in the environment?
+
+!!! Python interpreters
+    The python interpreter is also installed in the environment. This is because the python interpreter version is read from the `requires-python` field in the `pyproject.toml` file. This is used to determine the python version to install in the environment. This way pixi automatically manages/bootstraps the python interpreter for you similar as to a tool like `rye` can do.
```

Comment:
```suggestion
    The python interpreter is also installed in the environment. This is because the python interpreter version is read from the `requires-python` field in the `pyproject.toml` file. This is used to determine the python version to install in the environment. This way pixi automatically manages/bootstraps the python interpreter for you, so no more `brew`, `apt` or other system install steps.
```
Don't think we should mention `rye` here, as it is not that known, If you want to keep it in, i would propose to make it a link.

Reply:
just my 2 cents - I would keep it because it was a "hyped" tool in python world for a while so maybe we can win some attention by this.
but feel free to remove it